addons/test_mail/static/tests/activity_tests.js

```diff
@@ -1103,7 +1110,7 @@ QUnit.module("test_mail", {}, function () {
                 state: "planned",
                 activity_type_id: mailActivityTypeIds[2],
                 mail_template_ids: mailTemplateIds,
-                user_id: resUsersId1,
+                user_id: pyEnv.currentUserId,
```

Comment:
We don't need to change the user assigned in existing tests in the context of this PR.

Reply:
`resUserId1` doesn't have a partner (it makes the activity crash), and the test was searching for the first existing user instead of creating a user.